source/tutorial/simple-component.md

```diff
@@ -78,7 +80,9 @@ For the button to make the image show, we will need to add an action that change
 ...
 ```
 
-Clicking this button will send the action to the component:
+Clicking this button will send the action to the component.
+Ember will then go into the `actions` object and call the `imageShow` function.
```

Comment:
I believe the `actions` thingy is usually referred to as the `actions` hash, rather than the `actions` object. (I personally prefer thingy, though.)